cost/aws/burstable_instance_cloudwatch_credit_utilization/aws_burstable_instance_cloudwatch_credit_utilization.pt

```diff
@@ -0,0 +1,702 @@
+name "AWS Burstable Instance CloudWatch Utilization"
+rs_pt_ver 20180301
+type "policy"
+short_description "Gathers AWS CloudWatch CPU and Burst Credit data for instances on 30 day intervals. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/aws/burstable_instance_cloudwatch_credit_utilization/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+severity "low"
+category "Cost"
+
+###############################################################################
+# Permissions
+###############################################################################
+
+permission "perm_read_creds" do
+  actions   "rs_cm.show_sensitive","rs_cm.index_sensitive"
+  resources "rs_cm.credentials"
+end
+
+###############################################################################
+# Parameters
+###############################################################################
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses to notify"
+  description "Email addresses of the recipients you wish to notify when new incidents are created"
+end
+
+parameter "param_num_cpu_surplus_credits" do
+  type "number"
+  label "Number of Surplus Credits to alert on"
+  description "Number of CPU Surplus Credits to report on, Set to -1 to ignore cpu burst credits"
+  default -1
+  min_value -1
+end
+
+parameter "param_bool_cpu_burst_credit_balance" do
+  type "string"
+  label "Enable checking burst credit balance against max"
+  description "checks burst credit balance against max_earnable_credits"
+  default "false"
+  allowed_values "true", "false"
+end
+
+parameter "param_exclusion_tag_key" do
+  category "User Inputs"
+  label "Exclusion Tag Key"
+  description "Cloud native tag key to ignore instances. Format: Key:Value"
+  type "string"
+end
+
+parameter "param_take_action" do
+  category "User Inputs"
+  label "Action Direction"
+  description "Should this policy upsize or downsize"
+  type "string"
+  allowed_values "up","down"
```

Comment:
since the description has upsize and downsize  I think the allowed values should also use upsize and downsize.  Also, there should be a mention of what upsize and downsize means somewhere in the readme.